### ISSUE TITLE
[Proposal] - OpenCore Source Code Backups

### ISSUE THREAD

**Issue by 1alessandro1** [acidanthera/bugtracker#1254]
Goodmoring everyone, having seen what happened to youtube-dl github repository, (in case you didn't know, it was shut down a couple of days ago) I propose to sync the source code with other repositories such as `bitbucket`, `gitlab` etc - in order to have a backup plan in case something related to burocracy goes wrong (such as some crazy company pretending to sell macOS with OC bootloader inside - could bring some Apple's attention towards this project)

TL; DR
Backups are inexpensive, and they are very welcome in order to save the awesome work that has been done and will be done.

Thanks

**Comment by Andrey1970AppleLife** [comment]
The source code is in local repositories of all developers and can't be lost.
